engine-tests/src/test/java/org/terasology/rendering/cameras/PerspectiveCameraTest.java

```diff
@@ -55,24 +56,28 @@ public void testPerspectiveCamera() {
         camera.setOrientation(new Quat4f(45, .001f, .001f));
         camera.updateMatrices();
 
-        Matrix4f viewMatrix = new Matrix4f(5.0273395f, 0.0f, 0.0f, 0.0f
-                , 0.0f, 8.937492f, 0.0f, 0.0f
-                , 0.0f, 0.0f, -1.00004f, -0.20000401f
-                , 0.0f, 0.0f, -1.0f, 0.0f);
+        Matrix4f viewMatrix = new Matrix4f(-5.253E-1f, 0.000E+0f, 8.509E-1f, 0.000E+0f,
+            8.509E-4f, 1.000E+0f, 5.253E-4f, 0.000E+0f,
+            -8.509E-1f, 1.000E-3f, -5.253E-1f, 0.000E+0f,
+            -0.000E+0f, -0.000E+0f, -0.000E+0f, 1.000E+0f);
         Matrix4f projectionMatrix = new Matrix4f(5.0273395f, 0.0f, 0.0f, 0.0f
-                , 0.0f, 8.937492f, 0.0f, 0.0f
-                , 0.0f, 0.0f, -1.00004f, -0.20000401f
-                , 0.0f, 0.0f, -1.0f, 0.0f);
-        Matrix4f normViewMatrix = new Matrix4f(0.27303523f,0.0f,-0.962004f,-0.0f
-                ,-0.96200305f,0.0014514439f,-0.27303496f,-0.0f,
-                0.0013962948f,0.99999887f,3.962953f,-0.0f,
-                0.0f,0.0f,0.0f,1.0f);
+            , 0.0f, 8.937492f, 0.0f, 0.0f
+            , 0.0f, 0.0f, -1.00004f, -0.20000401f
+            , 0.0f, 0.0f, -1.0f, 0.0f);
+        Matrix4f normViewMatrix = new Matrix4f(-.525322f, 0.0f, 0.8509036f, -0.0f
+            , 8.509035E-4f, 0.9999995f, 5.25322E-4f, -0.0f,
+            -0.85090315f, 0.001f, -0.5253218f, -0.0f,
+            0.0f, 0.0f, 0.0f, 1.0f);
```

Comment:
Here we've got what looks to be the same values as in `viewMatrix`, _almost._

Are those the same numbers, but displayed a little differently due to binary-floating-point vs decimal representations? Or is the change of 0.01% significant? Could this have been a copy of viewMatrix instead?

I don't believe it's necessary to remove _all_ numbers from tests (in a math library, even!), but I'm failing to tell signal from noise here.

Reply:
On a second reading, I see this test _does_ have that camera.setOrientation with the 0.001 radian pitch and yaw, so maybe those tiny differences _are_ significant?